discord/gateway.py

```diff
@@ -535,18 +535,26 @@ async def request_sync(self, guild_ids):
         }
         await self.send_as_json(payload)
 
-    async def request_chunks(self, guild_id, query, limit, *, nonce=None):
+    async def request_chunks(self, guild_id, query, limit, *, user_ids=None, nonce=None):
```

Comment:
Might be worth seeing the remaining call sites due to the internal signature change.

```suggestion
    async def request_chunks(self, guild_id, query=None, *, limit, user_ids=None, nonce=None):
```